e2e-tests/README.md

```diff
@@ -0,0 +1,103 @@
+
+**Note:** there is no need to build an image if you are going to test some already-released version.
+
+## Building and testing the Operator
+
+There are scripts which build the image and run tests. Both building and testing
+needs some repository for the newly created docker images. If nothing is
+specified, scripts use Percona's experimental repository `perconalab/percona-server-mongodb-operator`, which
+obviously requires decent access rights to make a push.
+
+To specify your own repository for the Percona Server for MongoDB Operator docker image, you can use IMAGE environment variable:
+
+```
+export IMAGE=bob/my_repository_for_test_images:K8SPSMDB-372-fix-feature-X
+```
+
+Use the following script to build the image:
+
+```
+./e2e-tests/build
+```
+
+You can also build the image and run your cluster in one command:
+
+```
+./e2e-tests/build-and-run
+```
+
+Running all tests at once can be done with the following command:
+
+```
+./e2e-tests/run
+```
+
+(see how to configure the testing infrastructure [here](#using-environment-variables-to-customize-the-testing-process)).
+
+Tests can be executed one-by-one, usingthe following scripts (their names  should be self-explanatory):
+
+
+```
+./e2e-tests/init-deploy/run
+./e2e-tests/arbiter/run
+./e2e-tests/limits/run
+./e2e-tests/scaling/run
+./e2e-tests/demand-backup/run
+./e2e-tests/scheduled-backup/run
+./e2e-tests/storage/run
+./e2e-tests/self-healing/run
+./e2e-tests/operator-self-healing/run
+....
+```
+
+## Using environment variables to customize the testing process
+
+### Re-declaring default image names
+
+You can use environment variables to re-declare all default images used for testing. The
+full list of variables is the following one:
+
+* `IMAGE` - Percona Server for MongoDB Operator, `perconalab/percona-server-mongodb-operator:main` by default,
+* `IMAGE_MONGOD` - mongod, `perconalab/percona-server-mongodb-operator:main-mongod4.4` by default,
+* `IMAGE_PMM` - Percona Monitoring and Management (PMM) client, `perconalab/pmm-client:dev-latest` by default,
+* `IMAGE_BACKUP` - backup, `perconalab/percona-server-mongodb-operator:main-backup` by default,
+
+### Skipping backup tests on S3-compatible storage
+
+Making backups [on S3-compatible storage](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/backups.html#making-scheduled-backups) needs creating Secrets to have the access to the S3 buckets. There is an environment variable which allows to skip all tests which require such Secrets.
+
+```
+SKIP_BACKUPS_TO_AWS_GCP=1
```

Comment:
I am going to enable it by default, so please mention that it is enabled by default